defaults/main.yml

```diff
@@ -2,6 +2,7 @@
 
 golang:
   tarball: go1.6.linux-amd64.tar.gz
+  url: "{{ http://golang.org/dl/{{ golang.tarball }} }}"
```

Comment:
Perhaps this should be base_url instead of URL?

Eg.

```YML
golang:
  base_url: http://golang.org/dl/
  tarball: go1.6.linux-amd64.tar.gz
```